manifest.json

```diff
@@ -0,0 +1,430 @@
+      "sizes": "96x96"
+    },
+    {
+      "src": "app-icons/android/android-launchericon-72-72.png",
+      "sizes": "72x72"
+    },
+    {
+      "src": "app-icons/android/android-launchericon-48-48.png",
+      "sizes": "48x48"
+    },
+    {
+      "src": "app-icons/ios/ios-appicon-1024-1024.png",
+      "sizes": "1024x1024"
+    },
+    {
+      "src": "app-icons/ios/ios-appicon-180-180.png",
+      "sizes": "180x180"
+    },
+    {
+      "src": "app-icons/ios/ios-appicon-152-152.png",
+      "sizes": "152x152"
+    },
+    {
+      "src": "app-icons/ios/ios-appicon-120-120.png",
+      "sizes": "120x120"
+    },
+    {
+      "src": "app-icons/ios/ios-appicon-76-76.png",
+      "sizes": "76x76"
+    },
+    {
+      "src": "app-icons/ios/ios-launchimage-750-1334.png",
+      "sizes": "750x1334"
+    },
+    {
+      "src": "app-icons/ios/ios-launchimage-1334-750.png",
+      "sizes": "1334x750"
+    },
+    {
+      "src": "app-icons/ios/ios-launchimage-1242-2208.png",
+      "sizes": "1242x2208"
+    },
+    {
+      "src": "app-icons/ios/ios-launchimage-2208-1242.png",
+      "sizes": "2208x1242"
+    },
+    {
+      "src": "app-icons/ios/ios-launchimage-640-960.png",
+      "sizes": "640x960"
+    },
+    {
+      "src": "app-icons/ios/ios-launchimage-640-1136.png",
+      "sizes": "640x1136"
+    },
+    {
+      "src": "app-icons/ios/ios-launchimage-1536-2048.png",
+      "sizes": "1536x2048"
+    },
+    {
+      "src": "app-icons/ios/ios-launchimage-2048-1536.png",
+      "sizes": "2048x1536"
+    },
+    {
+      "src": "app-icons/ios/ios-launchimage-768-1024.png",
+      "sizes": "768x1024"
+    },
+    {
+      "src": "app-icons/ios/ios-launchimage-1024-768.png",
+      "sizes": "1024x768"
+    },
+    {
+      "src": "app-icons/chrome/chrome-extensionmanagementpage-48-48.png",
+      "sizes": "48x48"
+    },
+    {
+      "src": "app-icons/chrome/chrome-favicon-16-16.png",
+      "sizes": "16x16"
+    },
+    {
+      "src": "app-icons/chrome/chrome-installprocess-128-128.png",
+      "sizes": "128x128"
+    },
+    {
+      "src": "app-icons/firefox/firefox-marketplace-512-512.png",
+      "sizes": "512x512"
+    },
+    {
+      "src": "app-icons/firefox/firefox-marketplace-128-128.png",
+      "sizes": "128x128"
+    },
+    {
+      "src": "app-icons/firefox/firefox-general-256-256.png",
+      "sizes": "256x256"
+    },
+    {
+      "src": "app-icons/firefox/firefox-general-128-128.png",
+      "sizes": "128x128"
+    },
+    {
+      "src": "app-icons/firefox/firefox-general-90-90.png",
+      "sizes": "90x90"
+    },
+    {
+      "src": "app-icons/firefox/firefox-general-64-64.png",
+      "sizes": "64x64"
+    },
+    {
+      "src": "app-icons/firefox/firefox-general-48-48.png",
+      "sizes": "48x48"
+    },
+    {
+      "src": "app-icons/firefox/firefox-general-32-32.png",
+      "sizes": "32x32"
+    },
+    {
+      "src": "app-icons/firefox/firefox-general-16-16.png",
+      "sizes": "16x16"
```

Comment:
Do we really need all these images?

Reply:
They came from an icon generator which aimed to cover all bases. If you specify which icons you want to keep I can remove the rest?